tenable/io/v3/vm/scanners.py

```diff
@@ -143,7 +144,7 @@ def details(self, id: int) -> Dict:
             >>> scanner = tio.v3.vm.scanners.details(1)
             >>> pprint(scanner)
         """
-        return self._api.get(f"{id}").json()
+        return self._get(f"{id}")
```

Comment:
no need for an f-string here, just passing id is sufficient.

Reply:
Actually the API expects a string when I tried with https://qa-develop.cloud.aws.tenablesecurity.com hence kept the string
```  File "test.py", line 65, in get_scanners
    print(tio.v3.vm.scanners.details(50392291))
  File "d:\users\hashah.ctr\desktop\tenable\pytenable\tenable\io\v3\vm\scanners.py", line 156, in details
    return self._get(id)
  File "D:\Users\hashah.ctr\Desktop\tenable\venv\lib\site-packages\restfly-1.4.4-py3.7.egg\restfly\endpoint.py", line 129, in _get
  File "D:\Users\hashah.ctr\Desktop\tenable\venv\lib\site-packages\restfly-1.4.4-py3.7.egg\restfly\endpoint.py", line 80, in _req
TypeError: sequence item 1: expected str instance, int found
```